Librarian/index.js

```diff
@@ -1,67 +1,79 @@
-"use strict";
+require('babel/register');
 
-var Series = require('Librarian/Series')();
-var AddBook = require('Librarian/AddBook')();
-var InspectKindlize = require('Librarian/InspectKindlize')();
-var RepairImg = require('Librarian/RepairImg')();
-var AddASIN = require('Librarian/AddASIN')();
-var UpdateUrl = require('Librarian/UpdateUrl')();
-var SendStatus = require('Librarian/SendStatus')();
+const Series = require('Librarian/Series')();
+const AddBook = require('Librarian/AddBook')();
+const InspectKindlize = require('Librarian/InspectKindlize')();
+const RepairImg = require('Librarian/RepairImg')();
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).